### ISSUE TITLE
Documentation for streaming content

### ISSUE THREAD

**Issue by Don42** [QuietMisdreavus/twitter-rs#42]
There seems to be functionality to stream a users timeline, but I can figure out how to use it.
I haven't found the stream module in the documentation and no example using it.
Have I been looking in the wrong area?

**Comment by QuietMisdreavus** [comment]
Yeah, streams haven't made it to a full release. I doubt they'll ever make it to a full release, since they're [slated to be removed entirely this summer](https://blog.twitter.com/developer/en_us/topics/tools/2017/announcing-more-functionality-to-improve-customer-engagements-on-twitter.html). That's also why their docs aren't as fleshed-out as the rest of egg-mode - since i haven't made a release with them, i haven't made a docs pass over them.
